prov/verbs/src/fi_verbs.c

```diff
@@ -990,14 +1004,8 @@ fi_ibv_create_ep(const char *node, const char *service,
 
 	ret = rdma_create_ep(id, _rai, NULL, NULL);
 	if (ret) {
+		VERBS_INFO_ENO(FABRIC, "rdma_create_ep", errno);
 		ret = -errno;
-		if (ret == -ENOENT) {
-			FI_WARN(&fi_ibv_prov, FI_LOG_CORE,
-				"rdma_create_ep()-->ENOENT; "
-				"likely usnic bug, skipping verbs provider.\n");
-			ret = -FI_ENODATA;
-		}
-		FI_INFO(&fi_ibv_prov, FI_LOG_CORE, "Unable to create rdma_cm id\n");
```

Comment:
> If a valid verbs device is present alongside an usNIC device, are you saying rdma_create_ep would fail because of the usNIC device?

Yes, that's my (untested) belief.  But again, I'm not concerned about it because it will be a rare case with a simple workaround.

> For an active endpoint, I'm guessing this shouldn't happen since _rai would correspond to the valid verbs device.

Would it?  I don't think the filtering logic in #988 will filter usNIC devices when there are non-usNIC verbs devices present in the system.  There is definitely a case on the active side that prompted me to write 8b71d9d in the first place.

In any case, I would leave this PR (#1241) as-is w.r.t. this little code snippet.